src/components/button/button.vue

```diff
@@ -10,86 +10,109 @@ module.exports = {
     label: String,
     modifier: String,
     disabled: Boolean
+  },
+  computed: {
+    modifierClass() {
+      return this.modifier ? 'apos-button--' + this.modifier : false;
+    }
+  },
+  methods: {
+    click() {
+    	this.$emit('click');
+    }
   }
 }
 </script>
 
 <style lang="scss">
   .apos-button {
-    color: var(--neutral-eight);
-    background: var(--primary);
-    padding: 10px 20px; // re use this?
+    color: var(--text-primary);
+    background-color: var(--neutral-nine);
```

Comment:
I worry about us using specific color names like this directly. Below you use `--primary-button-hover`, which is semantic and can be reset on the theme level. I realize `neutral-nine` isn't the same as `grey-light-3`, but it's tied to an order (9th) that I'm not sure will be the same in an alternate theme (e.g., dark mode). Right?

Reply:
no, part of this refactor is that a theme's neutral scale must follow an order from most contrasting to least contrasting. we need to document that system but that is the only way we can do this without rewrapping lots of application specific variables (like have $neutral-two be some gray, then $button-hover: $neutral-two, then in dark mode re map $button-hover to $neutral-nine, for example. I started a branch of that version and it was quite hellish)